testify/test_case.py

```diff
@@ -195,6 +195,12 @@ def runnable_test_methods(self):
     def run(self):
         """Delegator method encapsulating the flow for executing a TestCase instance.
         """
+        # The TestResult constructor wants an actual method, which it inspects
+        # to determine the method name (and class name, so it must be a method
+        # and not a function!). self.run is as good a method as any.
+        test_case_result = TestResult(self.run)
+        test_case_result.start()
+        self.fire_event(self.EVENT_ON_RUN_TEST_CASE, test_case_result)
```

Comment:
Is this fake `.run` method necessary?  This used to be just so the sql reporter could know when to stop so I removed it when I removed the sql reporter.